File: app/src/main/java/omniex/nl/omniex/ui/app/order/shipping/OrderShippingView.java
package omniex.nl.omniex.ui.app.order.shipping;

import java.util.List;

import nl.omniex.omniexshopping.data.model.address.Address;
import nl.omniex.omniexshopping.ui.base.BaseView;

public interface OrderShippingView extends BaseView {

    void onShippingAddressesFetched(List<Address> addressList);
    void onGetShippingAddressesFailed();
    void onShippingAddressSet();
}

File: app/src/main/java/omniex/nl/omniex/ui/app/order/shipping/OrderShippingView.kt
package omniex.nl.omniex.ui.app.order.shipping

import omniex.nl.omniex.data.model.address.Address
import omniex.nl.omniex.ui.base.BaseView

interface OrderShippingView : BaseView {

    fun onShippingAddressesFetched(addressList: List<Address>)
    fun onGetShippingAddressesFailed()
    fun onShippingAddressSet()
}
